staff/luis-garrido/e-commerce-template/index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Tienduca</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+
+    <header>
+        <div id="logo"><img src="https://99designs-start-attachments.imgix.net/alchemy-pictures/2016%2F02%2F22%2F04%2F07%2F21%2F9757e437-5ec1-4378-804f-ca0f9567c110%2F380048_Widakk.png" height="80px"></div>
+        <div id="searchBar">
+            <input type="text" placeholder="Search..." />
+            <button type="submit" id="searchButton"></button>
+        </div>
+        <nav id="navHeader">
+            <div id="burguer">
+
+           </div>
+            <div id="twoItemsNav">
+                <div id="about">About</div>
+                <button id="contact">Contact</button>
+            </div>
+        </nav>
+    </header>
+
+    <main>
+        <section class="product">
+            <div id="productName"><h2>Producto</h2></div>
```

Comment:
I would use `header` instead of div here, something like `<header id="productName"><h2>Producto</h2></header>`for each product

Reply:
Fixed!